elastalert/schema.yaml

```diff
@@ -551,6 +551,11 @@ properties:
   twilio_message_service_sid: {type: string}
   twilio_use_copilot: {type: boolean}
 
+  ### Wechat
```

Comment:
There should be more schema definitions listed here since this alerter has a large number of input parameters.